package.json

```diff
@@ -12,10 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
     "main": "build/index.js",

```

Comment:
I can't run the branch from the dashboard code. I got "no module named ...imports".

Reply:
Hi Tom

Some day I may forget about this but I love open source too much. You need to add "type": "module" to package json. Or make it output as .mjs

Reply:
Have manged to run this locally and is now working